migrations/coin_policy_migration.go

```diff
@@ -0,0 +1,63 @@
+package migrations
+
+import (
+	"github.com/loomnetwork/go-loom"
+	ctypes "github.com/loomnetwork/go-loom/builtin/types/coin"
+	"github.com/loomnetwork/go-loom/plugin/contractpb"
+	"github.com/loomnetwork/loomchain"
+	"github.com/loomnetwork/loomchain/config"
+	"github.com/pkg/errors"
+)
+
+type (
+	Policy = ctypes.Policy
+)
+
+var (
+	policyKey = []byte("policy")
+)
+
+func GenerateCoinPolicyMigrationFn(cfg *config.Config) func(ctx *MigrationContext) error {
```

Comment:
The input parameters for the migration should be part of the migration tx, you shouldn't be reading the config here. We'll need to update the migration tx handler to support migration input parameters though.

Reply:
Done

Reply:
Migration Input Parameters are supplied as CLI arguments